core/cm4-tool/tool.c

```diff
@@ -54,76 +92,116 @@ int main(int argc, char *argv[])
 			example = 1;
 			break;
 
+		case 't':
+			terminal = 1;
+			termno = (int)strtol(optarg, NULL, 10);
+			break;
+
 		default:
 			usage = 1;
 			break;
 		}
 	}
 
-	if (usage || (file && example) || (!file && !example)) {
+	if (usage || (file && example) || (!file && !example && !terminal) ||
+			(terminal && (termno < 0 || termno > 4))) {
 		fprintf(stderr, "Shell tool for imxrt117x-cm4 driver. Usage:\n");
-		fprintf(stderr, "%s [-f file | -e]\n", argv[0]);
+		fprintf(stderr, "%s [-f file | -e | -t term]\n", argv[0]);
```

Comment:
```suggestion
		fprintf(stderr, "%s [-t term] <-f file | -e>\n", argv[0]);
```
If I am interpreting the conditions corretly, the help should be like this.

Reply:
Done, thanks